tests/src/Context/MinkContext.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\joinup\Context;
+
+use Behat\Mink\Driver\GoutteDriver;
+use Drupal\DrupalExtension\Context\MinkContext as DrupalExtensionMinkContext;
+
+/**
+ * Provides step definitions for interacting with Mink.
+ */
+class MinkContext extends DrupalExtensionMinkContext {
+
+  /**
+   * Attach File to field.
+   *
+   * @param string $field
+   *   Field name.
+   * @param string $path
+   *   File path.
+   */
+  public function attachFileToField($field, $path) {
```

Comment:
Hi @pfrenssen,
the reason for this is the following issue: https://github.com/minkphp/MinkSelenium2Driver/issues/187

When Selenium runs on a remote host (like it is now the case), it only seems to accept compressed files.